src/gui/snuffling.py

```diff
@@ -945,6 +947,18 @@ def rtrue(tr):
             trace_selector_arg = kwargs.pop('trace_selector', rtrue)
             trace_selector_viewer = self.get_viewer_trace_selector(mode)
 
+            if main_bandpass:
+                def apply_filters(traces):
+                    for tr in traces:
+                        if viewer.highpass is not None:
+                            tr.highpass(4, viewer.highpass)
+                        if viewer.lowpass is not None:
+                            tr.highpass(4, viewer.highpass)
```

Comment:
is this `tr.lowpass` and `viewer.lowpass`?

Reply:
whoops